plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentPluginImpl.kt

```diff
@@ -312,4 +352,14 @@ class LocationComponentPluginImpl : LocationComponentPlugin, LocationConsumer,
       deactivateLocationComponent()
     }
   }
-}
\ No newline at end of file
+}
+
+/**
+ * The default puck 2D style without showing bearing.
+ */
+lateinit var PUCK_2D_DEFAULT: LocationPuck2D
```

Comment:
Maybe we could expose the following two extension functions to create the puck presets? This way we don't need to expose new methods to the    LocationComponentPlugin interface and allow user to use our default puck styles.
```
fun LocationComponentPlugin.createDefault2DPuckWithBearing(context: Context)
fun LocationComponentPlugin.createDefault2DPuckWithoutBearing(context: Context)
```

Reply:
Sounds good to me.